### ISSUE TITLE
Support catalan geminate Ls

### ISSUE THREAD

**Issue by zipus** [mdn/content#26968]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform

### What specific section or headline is this issue about?

 language-specific case mapping rules

### What information was incorrect, unhelpful, or incomplete?

In lang="ca", when using text-transform:capitalize on a word containing "l·l" (https://en.wikipedia.org/wiki/Interpunct#Catalan) like word "cancel·lar" is converted to Cancel·Lar but should be Cancel·lar, so the dot should not be taken as the end of the word.

### What did you expect to see?

Cancel·lar

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/css/text-transform`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/css/text-transform/index.md
* Last commit: https://github.com/mdn/content/commit/abf155210d826c2584d23433eb2eac45a6669d23
* Document last modified: 2023-02-23T11:30:18.000Z

</details>

**Comment by Josh-Cena** [comment]
Yes, please file this to CSSWG. If it gets implemented we can document this.
